communication/slack-config/sig-release/config.yaml

```
@@ -8,3 +8,4 @@ channels:
   - name: release-management
     id: CJH2GBF7Y
   - name: release-notes
+  - name: release-leads
```

```suggestion
  - name: release-team-leads
```

You _may_ also need to update https://github.com/kubernetes/community/blob/master/communication/slack-config/restrictions.yaml

---

Tests seem happy without a change to `restrictions.yaml`. That file has the SIG Release one has having the pattern `"^release-"` which would still match `release-team-leads` right?